Got error "Cannot find module 'csstype'."

When upgrading from Material Ui next `1.0.0-beta.31` to `1.0.0-beta.46` I got the following error:

`error TS2307: Cannot find module 'csstype'.`

- [ x] I have searched the [issues](https://github.com/mui-org/material-ui/issues) of this repository and believe that this is not a duplicate.

## Expected Behavior

I should be able to compile the project with Typescript without errors.

## Current Behavior
It doesn't compile.

## Steps to Reproduce (for bugs)

Typescript react starter app that uses Material-UI beta.46.

## Context

I was able to fix this by adding a dev dependency on `"csstype": "^2.4.2",` so it seems like your project is missing that dependency?

## Your Environment

Project based on: https://github.com/Microsoft/TypeScript-React-Starter

| Tech         | Version |
|--------------|---------|
| Material-UI  |    1.0.0-beta.46     |
| React        |    16.0.27     |
| Typescript          |   2.5.3      |

--- a/packages/material-ui/package.json
+++ b/packages/material-ui/package.json
@@ -39,6 +39,7 @@
     "@types/react-transition-group": "^2.0.8",
     "brcast": "^3.0.1",
     "classnames": "^2.2.5",
+    "csstype": "^2.5.2",
     "debounce": "^1.1.0",
     "deepmerge": "^2.0.1",
     "dom-helpers": "^3.2.1",

